src/pages/index.js

```diff
@@ -0,0 +1,263 @@
+const imagePlus = document.getElementById("image-plus");
+
+imageLogo.src = logoSrc;
+imagePenBlack.src = penSrc;
+imagePenWhite.src = penWhiteSrc;
+imagePlus.src = plusSrc;
+
+const profileEditButton = document.querySelector(".profile__edit-btn");
+const editModal = document.querySelector("#edit-modal");
+
+const photoModal = document.querySelector("#photo-modal");
+const photoModalImage = photoModal.querySelector(".modal__photo");
+const photoModalCaption = photoModal.querySelector(".modal__description");
+
+const profileFormElement = document.forms["edit-profile"];
+const profileSaveBtn = profileFormElement.querySelector(".modal__save-button");
+
+const nameInput = profileFormElement.querySelector("#name");
+const jobInput = profileFormElement.querySelector("#description");
+
+const profileNameElement = document.querySelector(".profile__name");
+const profileJobElement = document.querySelector(".profile__description");
+
+api
+  .getAppInfo()
+  .then((result) => {
+    const savedCards = result[0];
+    const savedProfile = result[1];
+    imageAvatar.src = savedProfile["avatar"];
+    imageAvatar.alt = savedProfile["name"];
+    profileNameElement.textContent = savedProfile["name"];
+    profileJobElement.textContent = savedProfile["about"];
+    savedCards.forEach((card) => {
+      const cardEl = getCardElements(card);
+      cardsList.append(cardEl);
+    });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+function openModal(modal) {
+  modal.classList.add("modal_opened");
+  document.addEventListener("keydown", escapeListener);
+}
+
+function closeModal(modal) {
+  modal.classList.remove("modal_opened");
+  document.removeEventListener("keydown", escapeListener);
+}
+
+function escapeListener(evt) {
+  if (evt.key === "Escape") {
+    const modal = document.querySelector(".modal_opened");
+    closeModal(modal);
+  }
+}
+
+const modalList = document.querySelectorAll(".modal");
+modalList.forEach((modal) => {
+  modal.addEventListener("click", function (evt) {
+    if (evt.target.classList.contains("modal")) {
+      closeModal(modal);
+    }
+  });
+});
+
+const closeButtons = document.querySelectorAll(".modal__close-button");
+closeButtons.forEach((button) => {
+  const popup = button.closest(".modal");
+  button.addEventListener("click", (evt) => {
+    closeModal(popup);
+  });
+});
+
+profileEditButton.addEventListener("click", () => {
+  openModal(editModal);
+
+  nameInput.value = profileNameElement.textContent;
+  jobInput.value = profileJobElement.textContent;
+
+  disableButton(profileSaveBtn, settings);
+});
+
+function handleProfileFormSubmit(evt) {
+  function makeRequest() {
+    return api
+      .updateProfileInfo({ name: nameInput.value, about: jobInput.value })
+      .then((userData) => {
+        profileNameElement.textContent = userData["name"];
+        profileJobElement.textContent = userData["about"];
+        closeModal(editModal);
+      })
+      .catch((err) => console.error(err));
```

Comment:
`catch` is not needed if you use `handleSubmit` because it contains `catch` inside itself